diff --git a/node_modules/@docusaurus/core/lib/commands/start/watcher.js b/node_modules/@docusaurus/core/lib/commands/start/watcher.js
index 1637ecc..f917bdb 100644
--- a/node_modules/@docusaurus/core/lib/commands/start/watcher.js
+++ b/node_modules/@docusaurus/core/lib/commands/start/watcher.js
@@ -31,7 +31,12 @@ function watch(params, callback) {
         ignoreInitial: true,
         ...options,
     });
-    fsWatcher.on('all', (name, eventPath) => callback({ name, path: eventPath }));
+    fsWatcher.on('all', (name, eventPath) => {
+        if (/docs\\functions\\\w+\.md$/.test(eventPath)) return;
+        console.log("Reloading due to change of file: " + eventPath);
+
+        callback({ name, path: eventPath })
+    });
     return () => fsWatcher.close();
 }
 exports.watch = watch;
